### PR TITLE
Removed overrides from manifest file

### PR DESCRIPTION
**Pull request by n-boshnakov**
**What this PR does / why we need it**:
This PR removes the overrides from the manifest file and adds titles to the md files instead.
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```doc developer
NONE
```

### CODE REVIEW

**Review by Kristian-ZH — APPROVED** [review]
/lgtm

File: docs/development/cp_support_old.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+---
+title: Adding Support for a Cloud Provider (Legacy)
+draft: false
```
Comment (Kristian-ZH) [inline docs/development/cp_support_old.md:3]: I think that draft: false is the default value so you can just remove it

**Review by Kristian-ZH — APPROVED** [review]

**Review by himanshu-kun — APPROVED** [review]
/lgtm

**Pull request merged**